core/modules/nlohmann/json.hpp

```diff
@@ -0,0 +1,14031 @@
+/*
```

Comment:
In theory that may be the case. In practice this is just one header file. And adding anything into EUPS (which we'll probably are going to divorce from) is going to take at least an RFC and a lengthy process of waiting for the next "official" LSST release. Ask Nate for his experience with ANTLR4. So, I would rather keep a copy of this file in Qserv as a module.

Reply:
It's freaking 14,000 lines, probably same scale as all qserv headers combined. I suggest we do the right thing and move it to eups. I'd also be OK with git submodule but we do not support that (yet).

Reply:
Let's discuss this with @fritzm

Reply:
I agree with @andy-slac here -- since it is general purpose, third party, and of significant scale, we should probably package for eups for the time being.  I can assist with this.

Reply:
RFC-480...